slash/conf.py

```diff
@@ -51,6 +51,7 @@
         "subpath": "{context.session.id}/{context.test_id}/debug.log" // Doc("Path to write logs to under the root"),
         "session_subpath": "{context.session.id}/session.log",
         "errors_subpath": None // Doc("If set, this path will be used to record errors added in the session and/or tests"),
```

Comment:
Add a deprecation notice here pointing to the new config value (in the doc string)